loader/loader.go

```diff
@@ -421,7 +421,12 @@ func (l *Loader) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	defer cancel()
 
 	l.newFileJobQueue()
-	l.getMydumpMetadata()
+	if err := l.getMydumpMetadata(); err != nil {
```

Comment:
`loaderExitWithErrorCounter.WithLabelValues(l.cfg.Name).Inc()`
increase `loaderExitWithErrorCounter` in this code block?

Reply:
updated